src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/ApplicableComponentsAttribute.cs

```diff
@@ -1,10 +1,10 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using System;
-
 namespace Microsoft.DotNet.UpgradeAssistant
 {
+    using System;
```

Comment:
Convention for UA has been to have `using` statements outside namespaces.

Reply:
sounds good